packages/astro/src/@types/astro.ts

```diff
@@ -2184,6 +2184,259 @@ export interface AstroUserConfig {
 		 * For a complete overview, and to give feedback on this experimental API, see the [Server Islands RFC](https://github.com/withastro/roadmap/pull/963).
 		 */
 		serverIslands?: boolean;
+
+		/**
+		 * @docs
+		 * @name experimental.contentLayer
+		 * @type {boolean}
+		 * @default `false`
+		 * @version 4.16.0
+		 * @description
+		 *
+		 * The Content Layer API is a new way to handle content and data in Astro. It builds upon [content collections](https://docs.astro.build/en/guides/content-collections/), taking them beyond local files in `src/content` and allowing you to fetch content from anywhere, including remote APIs, or files anywhere in your project. As well as being more powerful, the Content Layer is designed to be more performant, helping sites scale to thousands of pages. Data is cached between builds and updated incrementally. Markdown parsing is also 5-10x faster, with similar scale reductions in memory. While the feature is experimental and subject to breaking changes, we invite you to try it today and let us know how it works for you.
```

Comment:
Docs comment re: terminology (with full understanding this can change in the experimental stage easily enough!) Sorry for the length!

I see in the RFC we still use "collection" and "entry". As I'm going through the text, it feels the most natural *to me* (outsider, thinking of explaining this feature) to refer to "content layer" in e.g. the following ways

- "use the Content Layer API"

- "collections powered by the Content Layer API"

- "migrating your existing content collections to use the Content Layer API" (vs migrating from content collections to content layer" <- this doesn't sound quite right, because implies that you don't still have collections")

- defining new (content?) collections (with the Content Layer API) <- this last part will be unnecessary once this is the only way. And, with all our marketing having gone into "Content collections" and being a "content-first" project, we may even *want* to keep the phrase content collection to still refer to the noun, irrespective of the new implementation.

"content layer" doesn't feel like it can replace "content collections" one-to-one. You don't have "different layers." You do have "a content layer" but that's not *instead* of having collections, and that's not the main part of this that the *user* interacts with: they make *collections* that have *entries*. I don't think "layer collections" feels very ergonomic, but that's really the closest analogy. :sweat_smile: 

So again, we don't have to have this all nailed down, but as I'm reviewing this I'll be suggesting what I think feels both clear and helpful to someone *using* this new API to power their collections!

 "The layer" may be the underlying interesting tech, but I really think "collection" is going to still be the main word that resonates with users the most. Once the layer is powering everything, no one is going to be thinking of a "content layer loader" when they write their collection schema, it's a loader they define *in their collection*, using `defineCollection()`. The word "layer" doesn't appear anywhere, and if we can start off closer to that kind of documentation, we'll be ahead of the game!

Reply:
Yeah, I think the right summary is that these new content collections are powered by the content layer API